### DISCUSSION TITLE
Nuxt 3.1.1 way to big build size

### DISCUSSION THREAD

**Discussion by melokki** [nuxt/nuxt#18580] [Questions] [answered]
Hello nuxt

I've just updated a project to nuxt 3.1.1 and I saw that when I am trying to publish my project to cloudflare workers, the build is way way too large compared to nuxt 3.0.0. Because of this the cloudflare deployment is rejected.
For comparison:
nuxt 3.0.0 build size:  
`Σ Total size: 845 kB (230 kB gzip) ` 
`Total Upload: 1172.89 KiB / gzip: 265.28 KiB`

nuxt 3.1.1 build size: 
`Σ Total size: 4.95 MB (1.88 MB gzip)`
`Total Upload: 7869.08 KiB / gzip: 2067.40 KiB`

resulting in cloudflare error

```
▲ [WARNING] Here are the 1 largest dependencies included in your script:

  - .output/server/index.mjs - 7867.06 KiB
  If these are unnecessary, consider removing them
```

From where comes this big difference?
It's the same project, nothing changed in it

I don't understand, is this normal?

**Comment by TheAlexLichter** [comment, marked as answer]
Should be fixed when upgrading Nitro. See https://github.com/nuxt/nuxt/issues/18608#issuecomment-1408533307

> Please upgrade Nuxt and Nitro using `npx nuxi upgrade --force`.
> 
> Nitropack `2.1.1` ([unjs/nitro#893](https://github.com/unjs/nitro/pull/893)) is likely to resolve it.

**Reply by melokki** [reply]
seems to be fixed indeed. thank you very much :)
